deepspeed/inference/config.py

```diff
@@ -152,6 +153,18 @@ class DeepSpeedInferenceConfig(DeepSpeedConfigModel):
     can run faster using the graph replay method.
     """
 
+    use_triton: bool = False
+    """
+    Use this flag to use triton kernels for inference ops.
+    """
+
+    triton_autotune: bool = False
```

Comment:
any reason why this should be false by default?

Reply:
if turned on, users have to pay for the large cost for the initial triton autotune.